File: backend-api/reservation-api/src/main/java/com/hcmute/reservation/model/dto/config/SystemConfigDTO.java
package com.hcmute.reservation.model.dto.config;

import lombok.Data;
import jakarta.validation.constraints.NotBlank;

@Data
public class SystemConfigDTO {
    @NotBlank(message = "Config key không được để trống")
    private String configKey;

    @NotBlank(message = "Config value không được để trống")
    private String configValue;

    private String description;
}

File: desktop-staff/reservation-winforms/DTO/config/SystemConfigDTO.cs
using Newtonsoft.Json;

namespace reservation_winforms.DTO.config
{
    public class SystemConfigDTO
    {
        [JsonProperty("configKey")]
        public string ConfigKey { get; set; }

        [JsonProperty("configValue")]
        public string ConfigValue { get; set; }

        [JsonProperty("description")]
        public string Description { get; set; }
    }
}